### DISCUSSION TITLE
Docker image source code request

### DISCUSSION THREAD

**Discussion by jackthenewbie** [anasty17/mirror-leech-telegram-bot#1605] [Q&A]
In dockerfile i can see: 
```dockerfile
FROM anasty17/mltb:latest
```
Apparently (could be docker cache image, now solved but not sure), the error occur because mltb:latest doesn't have rclone installed. So I have to add it manually
```dockerfile
FROM anasty17/mltb:latest

WORKDIR /usr/src/app
RUN chmod 777 /usr/src/app && curl https://rclone.org/install.sh | bash
COPY requirements.txt .
RUN pip3 install --no-cache-dir -r requirements.txt

COPY . .

CMD ["bash", "start.sh"]
```
Well I hope that my finding useful for someone.
Also can i request what your dockerfile of anasty17/mltb looks like? I want to tweak some stuff.

**Comment by anasty17** [comment]
How rclone not installed!! We all using it!

**Comment by anasty17** [comment]
> In dockerfile i can see:
> 
> ```dockerfile
> FROM anasty17/mltb:latest
> ```
> 
> Apparently (could be docker cache image, now solved but not sure), the error occur because mltb:latest doesn't have rclone installed. So I have to add it manually
> 
> ```dockerfile
> FROM anasty17/mltb:latest
> 
> WORKDIR /usr/src/app
> RUN chmod 777 /usr/src/app && curl https://rclone.org/install.sh | bash
> COPY requirements.txt .
> RUN pip3 install --no-cache-dir -r requirements.txt
> 
> COPY . .
> 
> CMD ["bash", "start.sh"]
> ```
> 
> Well I hope that my finding useful for someone. Also can i request what your dockerfile of anasty17/mltb looks like? I want to tweak some stuff.

I don't understand why someone will try to get the docker code?! If its hidden then its not available for public! POINT!
